server/db/driver/pg/internal/orders.go

```diff
@@ -68,11 +72,9 @@ const (
 	SelectOrderPreimage = `SELECT preimage FROM %s WHERE oid = $1;`
 
 	// SelectOrderCoinIDs retrieves the order id, sell flag, and coins for all
-	// orders in a table with one of the given statuses. Note that this includes
-	// all order statuses.
+	// orders in a certain table.
 	SelectOrderCoinIDs = `SELECT oid, sell, coins
-		FROM %s
-		WHERE type = ANY($1);`
```

Comment:
I'm pretty sure this was from a point when cancel orders were in the same table as trade orders (market and limit).